scraper/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Scrape {
+    pub sets: Vec<Set>,
+    pub types: Vec<String>,
+    pub cards: Vec<Card>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Set {
+    pub name: String,
+    pub card_indices: Vec<usize>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum Cost {
+    None,
+    Copper(u8),
```

Comment:
Should this be Copper? I think the term used in manual is 'worth' which I'm not a big fan of.

Reply:
For now I believe `Copper` is the smallest existing unit of money and it makes sense to express costs in terms of it. I need to learn about the other expansions before I can make a good suggestion, for now Copper seems reasonable.